dev/plugins/hu.elte.txtuml.api.stdlib/src/hu/elte/txtuml/api/stdlib/timers/Timer.xtxtuml

```diff
@@ -28,18 +35,38 @@
 	 *            the signal which is to be sent after the delay
 	 * @param millisecs
 	 *            the time in millisecs to wait before sending the signal
-	 * @return a timer instance to manage this delayed send operation before it
+	 * @return a handle object to manage this delayed send operation before it
 	 *         happens
 	 */
-	static Timer start(ModelClass targetObj, Signal signal, int millisecs) {
-		return new TimerImpl(targetObj, signal, millisecs);
+	public static external-body Timer start(Signal sig, ModelClass targetObj, int millisecs) {
```

Comment:
We DO allow `Signal` and `ModelClass` as parameter types.

`Timer(Signal, ModelClass, int)` is external to force the user to use the static method `Timer.start(Signal, ModelClass, int)` which shows better that we are simply sending the signal to the model object like we do it in the case of the similar `Action.send(Signal, ModelClass)` method. The only difference is that in case of timers, we add a delay and get a handle object as a result which can be used to manipulate this delayed send action until it is executed.